src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -823,7 +823,8 @@ export default async ({ req, res, log }) => {
         case 'json':
             return res.json({"type": "This is a JSON response"});
         case 'binary':
-            return res.binary(InputFile.fromPath('/path/to/file', 'filename'));
+            const bytes = await fs.readFile('/path/to/file');
```

Comment:
Let's make sure we're consistent. If it's `file.png` for the others, let's use that here too.